tests/AutoReview/CiConfigurationTest.php

```diff
@@ -52,9 +52,11 @@ public function testTestJobsRunOnEachPhp(): void
             self::generateMinorVersionsRange($supportedMinPhp, $supportedMaxPhp)
         );
 
+        static::assertTrue(\count($supportedVersions) > 0);
```

Comment:
Prevents PHPStan from reporting that `end($supportedVersions)` may return `false`.